src/format/mod.rs

```diff
@@ -43,9 +34,16 @@ impl<'a> InputElement<'a> {
 
     #[inline]
     pub fn is_required_type_of_var<T: ?Sized + Any>(&self, _var: &T) -> bool {
+        // Since we removed type checking, all placeholders accept any type
+        // The compiler will enforce type compatibility anyway
+        true
+    }
```

Comment:
Here `#[deprecated]` was needed in this function. And, we need to refactor the code for not need to call to this.